### ISSUE TITLE
[@types/ember bug] - should not expose private makeArray from @ember/array

### ISSUE THREAD

**Issue by mike-north** [typed-ember/ember-cli-typescript#292]
### Which package(s) does this problem pertain to?
  - [x] @types/ember

### What are instructions we can follow to reproduce the issue?
```ts
import { makeArray } from '@ember/array';

```

### Now about that bug. What did you expect to see?
I expected an error, indicating that there is no `makeArray` exported from the `@ember/array` package.

### What happened instead?
I'm able to import it, with no sign that this is a private thing I shouldn't use
